src/main/java/com/github/onsdigital/logging/layouts/TextLayout.java

```diff
@@ -27,8 +27,9 @@
 
     @Override
     public String doLayout(ILoggingEvent event) {
+
         StringBuilder sb = new StringBuilder();
-        sb.append(tertiaryColour(event, "[dp-logging]"))
+        sb.append(namespace(event, "[dp-logging]"))
```

Comment:
should `namespace` be called `styleNamespace` to be consistent with other methods?